tests/unit/helm/api-deployment_test.go

```diff
@@ -850,6 +844,74 @@ func (s *deploymentApiTemplateTest) TestContainerReadinessProbe() {
 	}
 }
 
+func (s *deploymentApiTemplateTest) TestContainerStartupProbe() {
+	testCases := []struct {
+		name     string
+		values   map[string]string
+		expected func(probe *corev1.Probe)
+	}{
+		{
+			"defaultValues",
+			nil,
+			func(probe *corev1.Probe) {
+				expectedProbeJSON := `{
+          "httpGet": {
+            "path": "/health/",
+            "port": "teams-api"
+          },
+		  "failureThreshold": 5,
+		  "periodSeconds": 5,
+          "timeoutSeconds": 5
+        }`
```

Comment:
The formatting here looks strange. Maybe it should be indented?

```suggestion
				expectedProbeJSON := `{
          "httpGet": {
            "path": "/health/",
            "port": "teams-api"
          },
		      "failureThreshold": 5,
		      "periodSeconds": 5,
          "timeoutSeconds": 5
        }`
```

Reply:
Oh, I think this is tabs vs spaces. Will fix.